azerothcore/azerothcore-wotlk
# fix(DB/Quest) add missing quest to brewfest npc's

## CHANGES PROPOSED:
-  Add quest "Catch the Wild Wolpertinger!" to both Goldark Snipehunter and Glodrak Huntsniper



## ISSUES ADDRESSED:
- Closes none



## TESTS PERFORMED:
- apply sql and restart server
- .event start 24
- checked that both Goldark Snipehunter and Glodrak Huntsniper now provide the quest.
- checked that you only get one Stunned Wolpertinger per use of the Wolpertinger Net
- checked that that you can complete the quest




## HOW TO TEST THE CHANGES:

- apply sql and restart server
- .event start 24
- check that both (.go c 88867) Goldark Snipehunter and (.go c 88932) Glodrak Huntsniper now provide the quest.
- check that you only get one Stunned Wolpertinger per use of the Wolpertinger Net
- check that that you can complete the quest

## KNOWN ISSUES AND TODO LIST:

- [x]  Seem when you use the "Wolpertinger Net" you receive two of the "Stunned Wolpertinger" when it should only be one.



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
